src/components/section/splash/SplashTitle.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export const SplashTitle = styled.p`
```

Comment:
as it's a title we might use something semantically more accurate (one of the HTML headline tags, maybe h3?)